shared/src/lib/knex-bigquery/index.js

```diff
@@ -0,0 +1,141 @@
+// ORIGINAL COPYRIGHT FOR THIS FILE CAN BE FOUND IN THE LICENSE FILE IN THIS FOLDER
+/* eslint-disable */
+const Promise = require('bluebird');
+const Client = require('knex/lib/client');
+
+const SchemaCompiler = require('./schema/compiler').default
+const QueryCompiler = require('./query/querycompiler').default
+const ColumnCompiler = require('./schema/columncompiler').default
+const TableCompiler = require('./schema/tablecompiler').default
+/* eslint-enable */
+
+
+class BigQueryClient extends Client {
+  constructor(config) {
+    super(config);
+  }
+
+  schemaCompiler() {
+    return new SchemaCompiler(this, ...arguments);
+  }
+
+  queryCompiler(builder, formatter) {
```

Comment:
This is more a linter thing that I'm surprised it didn't cry, can you less indent (dedent?) these